cadc-util/src/test/resources/SupportedDomains.properties

```diff
@@ -0,0 +1,2 @@
+# domains used for generating set of cookies in SSOCookieCredential
+domains = www.canfar.phys.uvic.ca www.cadc.hia.nrc.gc.ca www.ccda.iha.cnrc.gc.ca www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca
```

Comment:
Maybe mention that the expected value for the domains key is a space separated list of domains.

Also, shouldn't the domains in this list omit the www?  Better ask Dustin about that.